lib/commons/aria/get-owner-virtual.js

```diff
@@ -0,0 +1,54 @@
+/* global aria, dom */
+
+/**
+ * Get owner element of a provided virtualNode
+ * @param {Object} virtualNode virtual node
+ * @returns {HTMLElement | undefined}
+ */
+aria.getOwnerVirtual = function getOwnerVirtual(virtualNode) {
+	return getAriaOwner(virtualNode) || getOwner(virtualNode);
```

Comment:
Before doing this, can you test that the thing doesn't have role=presentation or none. Those elements can't be owned.